docs/source/datasets_others.rst

```diff
@@ -0,0 +1,26 @@
+.. role:: hidden
```

Comment:
Same here. Please have a look at my comment above.

Reply:
@akihironitta Thank you. I will remove this one as well.